modules/pel/peltool/README.md

```diff
@@ -0,0 +1,67 @@
+
+##### List PELs based on severity and only options
+
+- Get PEls including PELs with severity as Informational: `peltool.py -l -S Informational`
+- **Note:** User can filter the PELs by severity value from: `{Informational, Recovered, Predictive, Unrecoverable, Critical, Diagnostic, Symptom}`. Can provide multiple values e.g. `peltool.py -l -S Informational Critical`
+- Get hidden PEls only: `peltool.py -lHO`
+- Get critical PEls only: `peltool.py -lO -S Critical`
+
+
+#### Show PEL Count
+- Get PEL count : `peltool.py -a` (By default serviceable PELs are listed)
+   **Note:** For non-BMC environment user must pass the PEL directory e.g. `peltool.py -p <pel_dir> -n`
+- Get PEL count including serviceable PELs : `peltool.py -ns`
+- Get PEL count including non-serviceable PELs: `peltool.py -nN`
+- Get PEL count including hidden PELs: `peltool.py -nH`
+- Get PEL count including terminating PELs: `peltool.py -nt`
+
+##### Show PEL count based on severity and only options
+
+- Get PEL count including PELs with severity as Informational: `peltool.py -n -S Informational`
+- **Note:** User can filter the PELs by severity value from: `{Informational, Recovered, Predictive, Unrecoverable, Critical, Diagnostic, Symptom}`. Can provide multiple values e.g. `peltool.py -n -S Informational Critical`
+- Get hidden PEls count only: `peltool.py -nHO`
+- Get critical PEls count only: `peltool.py -nO -S Critical`
+
+#### Display all PEL data
+- Get PEL count : `peltool.py -a` (By default serviceable PELs are listed)
+  **Note:** For non-BMC environment user must pass the PEL directory e.g. `peltool.py -p <pel_dir> -a`
+- Get all PEL data including serviceable PELs : `peltool.py -as`
+- Get all PEL data including non-serviceable PELs: `peltool.py -aN`
+- Get all PEL data including hidden PELs: `peltool.py -aH`
+- Get all PEL data including terminating PELs: `peltool.py -at`
+
+##### Show PEL count based on severity and only options
+- Get all PEL data including PELs with severity as Informational: `peltool.py -a -S Informational`
+- **Note:** User can filter the PELs by severity value from: `{Informational, Recovered, Predictive, Unrecoverable, Critical, Diagnostic, Symptom}`. Can provide multiple values e.g. `peltool.py -a -S Informational Critical`
+- Get hidden PEls data only: `peltool.py -aHO`
+- Get critical PEls data only: `peltool.py -aO -S Critical`
+
+#### Delete PELs
+- Delete single PEL:  `peltool.py -d <pel_id>`
+- Delete all PELs: `peltool.py -D`
+
+#### Get PELs based on ID type
+- Get PEL data based entry ID: `peltool.py -I <entry_id>`
+   **Note:** For non-BMC environment user must pass the PEL directory e.g. `peltool.py -p <pel_dir> -I <entry_id>`
+- Get PEL data based on BMC event log ID: `peltool.py —bmc-id <BMC_event_log_id>`
+- Get PELs based on platform log ID: `peltool.py —plid <platform log ID>`
+- Get PELs based on System reference code: `peltool.py —src <system_reference_code>`
```

Comment:
`<system_reference_code>` -> `<System_Seference_Code>`

and please add details about beautify of the `--src` option i.e This option could used to filter firmware, subsystem, component level PELs.